tilelang/language/__init__.py

```
@@ -64,7 +64,7 @@
     cumsum,  # noqa: F401
     finalize_reducer,  # noqa: F401
 )
-from .print import print  # noqa: F401
+from .print import print, device_assert  # noqa: F401
```

Ruff (0.14.1)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)